spec/importers/californica_mapper_spec.rb

```diff
@@ -17,7 +17,8 @@
       "Format.medium" => "1 photograph",
       "Name.repository" => "University of California, Los Angeles. $b Library Special Collections",
       "Description.caption" => "This example does not have a caption.",
-      "masterImageName" => "clusc_1_1_00010432a.tif" }
+      "masterImageName" => "clusc_1_1_00010432a.tif",
+      "Coverage.geographic" => "Los Angeles (Calif.)" }
```

Comment:
should we add `:location` to https://github.com/UCLALibrary/californica/blob/dbd7149720e7d73576946ff53c85dc7ec0dc99aa/spec/importers/californica_mapper_spec.rb#L47-L56 as well?